app/Controllers/feedController.php

```diff
@@ -640,7 +640,7 @@ public static function actualizeFeeds(?int $feed_id = null, ?string $feed_url =
 			}
 			unset($entries);
 
-			if (rand(0, 30) === 1) {	// Remove old entries once in 30.
+			if (mt_rand(0, 30) === 1) {	// Remove old entries once in 30.
```

Comment:
Please leave out those `rand` changes. They are unrelated to upgrading to a newer syntax.